projects/pgai/pgai/vectorizer/vectorizer.py

```diff
@@ -979,6 +979,7 @@ async def _load_copy_types(self, conn: AsyncConnection) -> None:
                 and k.relname operator(pg_catalog.=) %s
                 and a.attname operator(pg_catalog.!=) 'embedding_uuid'
                 and a.attnum operator(pg_catalog.>) 0
+                and a.attnotnull = true
```

Comment:
We should use LIMIT rather than `attnotnull`. A person could add a `not null` column with a default value, and it should work just fine. So, grab the first N columns ordered by `attnum`.